coalib/results/Diff.py

```diff
@@ -278,18 +276,37 @@ def modified(self):
         return result
 
     @property
+    def modified(self):
+        """
+        Calculates the modified file, after applying the Diff to the original.
+        Also adds linebreaks at the end of each line (except eof)
+        """
+        eof_linebreak = False
+
+        if self._file and self._file[-1].endswith('\n'):
+            eof_linebreak = True
+
+        file = [line.rstrip('\n') for line in self._modified()]
+
+        if eof_linebreak:
+            return self._add_linebreaks(file)
```

Comment:
> Since the "eof-state" (i.e. no new line at the end of the file) should be kept, the \n now be deleted.

That's the question, do we wanna keep that? I'm for abandoning the EOF-state, the user needs to change the new last line if he wants to keep the old no-\n-at-EOF-state. As it's in fact a change of that line.